### ISSUE TITLE
Loading DiCoM Image

### ISSUE THREAD

**Issue by AkshaY2039** [ben-heil/Data-Science-Bowl-2017#1]
How does numpy.load() help in your Utility functions to read .dcm file. Won't there be tags

**Comment by ben-heil** [comment]
You're correct. It's been a few years, but if I remember correctly I wrote the DICOM files in my dataset to image files or .npy for ease of use.

For advice on how to do the same, [this blog post](https://pyscience.wordpress.com/2014/09/08/dicom-in-python-importing-medical-image-data-into-numpy-with-pydicom-and-vtk/) might help
